src/Layout/Header/Tools/__tests__/index.spec.tsx

```diff
@@ -66,3 +89,13 @@ describe('Page header tools', () => {
   });
 
 });
+
+function createStore(cheCliTool: string): Store {
+  return new FakeStoreBuilder()
+    .withBranding({
+      configuration: {
+        cheCliTool
+      }
+    } as any)
```

Comment:
Try to avoid using type `any` whenever it's possible.

```suggestion
    .withBranding({
      configuration: {
        cheCliTool
      } as BrandingData
    })
```

Reply:
I have done it